cosmovisor/README.md

```diff
@@ -30,6 +30,28 @@ To install the latest version of `cosmovisor`, run the following command:
 go install github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor@latest
 ```
 
+To install a previous version, you can specify the version. IMPORTANT: Chains that use Cosmos-SDK v0.42.x MUST use Cosmovisor v0.1.0
+
+```
+go install github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor@v0.1.0
+```
+
+It is currently not possible to confirm the version of cosmovisor but will be included in Cosmovisor v1.1.0.
```

Comment:
`v1.0.0` already supports version (`cosmovisor version`). There is also `cosmovisor help`.